kubernetes/setup.cfg

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length=99
```

Comment:
this file is unnecessary. Could you remove it?

Reply:
Fixed.